api_docs.md

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
As you probably saw, I never got around to doing much of it on eklatz, but it would cool one day.